src/main/java/io/beanmapper/core/inspector/CombinedPropertyAccessor.java

```diff
@@ -127,4 +127,14 @@ public Method getReadMethod() {
     public Method getWriteMethod() {
         return methodAccessor != null ? methodAccessor.getWriteMethod() : null;
     }
+
+    /**
+     * {@inheritDoc}
+     */
+    @Override
+    public <A extends Annotation> boolean isAnnotationPresent(Class<A> annotationClass) {
+        if (this.fieldAccessor != null && this.fieldAccessor.isAnnotationPresent(annotationClass))
```

Comment:
`    @Override
    public <A extends Annotation> boolean isAnnotationPresent(Class<A> annotationClass) {
        return (this.fieldAccessor != null && this.fieldAccessor.isAnnotationPresent(annotationClass))
                || (this.methodAccessor != null && this.methodAccessor.isAnnotationPresent(annotationClass));
    }`